lib/simp/beaker_helpers.rb

```diff
@@ -584,7 +606,7 @@ def rhel_rhsm_subscribe(sut, *opts)
   end
 
   def sosreport(sut, dest='sosreports')
-    sut.install_package('sos')
+    on(sut, 'puppet resource package simp-release-community source="https://download.simp-project.com/simp-release-community.rpm" ensure=latest provider=rpm')
```

Comment:
- sos is in the base CentOS package for el7.  This rpm provides yum repos for epel, postgresql, puppet, and simp (https://download.simp-project.com/simp/yum...).  Why is this needed?
- If simp-beaker-helpers was used on an isolated network, there would be no way to substitute the correct url for the simp-release-community.rpm.